src/base-apis.js

```diff
@@ -1888,6 +1902,25 @@ MatrixBaseApis.prototype.getThirdpartyUser = function(protocol, params) {
     );
 };
 
+MatrixBaseApis.prototype.getTerms = function(serviceType, baseUrl, accessToken) {
```

Comment:
accessToken is not used

Reply:
I removed this as the GET endpoint doesn't require auth (at least that's what I put in MSC2140). I forgot to remove the param though.